.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug report
+about: Create a report to help us improve
+---
+
+<!-- Before creating a bug report, try disabling browser extensions to see if the bug is still present. -->
+
+<!-- If you're having trouble updating your profile, it is likely because you logged in separately with GitHub & Twitter. Please check if this is the case before creating a bug report, and email yo@dev.to so we can merge your accounts. -->
+
+**Describe the bug**
+
+<!-- A clear and concise description of what the bug is. -->
+
+**To Reproduce**
+
+<!-- Steps to reproduce the behavior: -->
+
+<!-- 1. Go to '...' -->
+<!-- 2. Click on '....' -->
+<!-- 3. Scroll down to '....' -->
+<!-- 4. See error -->
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+**Screenshots**
+
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+**Desktop (please complete the following information):**
+
+- OS, version:
+- Browser, version:
+
+**Smartphone (please complete the following information):**
+
+- Device:
+- OS, version:
+- Browser, version:
```

Comment:
I think we can combine these two sections, and keep the contents of the smartphone section. We could even ask the user to copy + paste their browser info from https://www.whatsmybrowser.org/.

```
**Device information:**

- [What's My Browser URL](https://www.whatsmybrowser.org/):

OR

- Device:
- OS & Version:
- Browser, version:
```